packages/components/src/components/primitives/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { Dropdown } from '.';
+
+export default {
+  component: Dropdown,
+  title: 'Components/Primitives/Dropdown',
+  argTypes: {},
+} as Meta;
+
```

Comment:
please add another story with the menu opened by default

Reply:
Done